Nawak (#4)
alexfauquette/material-ui-x: docs/data/data-grid/accessibility/accessibility.md

## Guidelines

> it is a above dec e.g. to (I assume show errors)

Author: ```suggestion
it's preceding dec for example to (I assume show errors)
```
